src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -390,6 +392,8 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
 
                 restoreTime.Stop();
 
+                await UpdateLastAccessTime(cacheFile);
```

Comment:
This probably needs to be above the `restoreTime.Stop()` call.

Reply:
Done